dali/driver/hid.py

```diff
@@ -714,40 +723,72 @@ def __init__(self, *args, **kwargs):
         self._response_available = asyncio.Event()
         self._response = None
 
+    def _initialise_device(self):
```

Comment:
when is this method called ?

Reply:
Line 143 self._initialise_device(), it overrides hid class method.

Reply:
please make a call to super()._initialise_device() and don't forget to set connected